src/map/status.c

```diff
@@ -2188,6 +2188,14 @@ bool status_check_skilluse(struct block_list *src, struct block_list *target, ui
 				if( tsc->option&hide_flag && !status_has_mode(status,MD_DETECTOR))
 					return false;
 			}
+			if (status_get_race2(target) == RC2_GVG || status_get_race2(target) == RC2_BATTLEFIELD) { // Skills disabled in GvG and Battleground against special racial grouped monsters
```

Comment:
The comment sounds a little misleading, since it is making it sound like it would be mapflag related, but this is a race only condition.
Also would it no be better to move it up to ~1996 where we have a switch by skill id already and just check the target race there?